Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Need devel version cause we need /usr/include/cudnn.h 
 # for compiling libctc_decoder_with_kenlm.so
-FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04
```

Comment:
Hm, is this going to be a problem for people using this Dockerfile to train? We probably need 10.0 and 10.1 side by side.

Reply:
Why would it be a problem ?

Reply:
Doesn't training with 1.15 require the older version and deps, like NCCL?

Reply:
That's not completely false, I guess we can't keep one dockerfile for everything?

Reply:
Yeah, would probably be better to have a simpler Dockerfile with just training stuff without building the world from scratch.